Driver/Driver/Extenison/Reusable.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Reusable.swift
+//  AfetYönetimSistemi
+//
+//  Created by Fatih Doğan on 14.02.2023.
+//
+
+import Foundation
+
+protocol Reusable {
+  static var identifier: String {get}
+}
+
+extension Reusable {
+  static var identifier: String {
+    return String(describing: self)
```

Comment:
If you write one line code while returning something, you don't need it no longer. Remove **return** here.